executor/aggregate.go

```diff
@@ -143,37 +143,7 @@ func (e *HashAggExec) execute(ctx context.Context) (err error) {
 
 // Next implements the Executor Next interface.
 func (e *HashAggExec) Next(ctx context.Context) (Row, error) {
```

Comment:
we can more `Next` function totally now.

Reply:
removed